packit/config/job_config.py

```diff
@@ -270,6 +270,11 @@ def get_default_jobs() -> List[Dict]:
     # no one will mutate the default jobs (hello tests)
     return deepcopy(
         [
+            {
+                "job": "copr_build",
+                "trigger": "pull_request",
+                "metadata": {"targets": "fedora-stable"},
```

Comment:
do we want to have here `fedora-stable` or `fedora-all`?

Reply:
should be the same as tests, so fedora-stable :)